hashicorp/web-unified-docs
# chore: remove Waypoint content

# Changes

- Remove waypoint content from code base to prep for education engineer test. This includes:
  - Removing waypoint from the `content` dir
  - Removing waypoint from the `public/assets` dir

- [Asana task](https://app.asana.com/0/1207899860738460/1208561647881342/f)

# Testing

1. Make a curl request or use Postman to send a `GET` request 
```
curl https://web-presence-experimental-docs.vercel.app/api/content/waypoint/nav-data/latest/docs
```
1. You should receive a valid response, such as this:
<img width="1143" alt="Screenshot 2024-10-16 at 16 45 39" src="https://github.com/user-attachments/assets/6d32b36f-1843-48e3-8d23-eb737abe021e">


1. Make a curl request or use Postman to send a `GET` request
```
curl https://web-presence-experimental-docs-git-heat-chorer-6a2d83-hashicorp.vercel.app/api/content/waypoint/nav-data/latest/docs
```
1. You should receive a response of `Not Found`
